src/ctia/stores/atom/common.clj

```diff
@@ -82,15 +82,14 @@
      filter-map :- {s/Any s/Any}]
 
     (when-not (empty? filter-map)
-      (into []
-            (filter (fn [model]
-                      (every? (fn [[k v]]
-                                (let [found-v (if (sequential? k)
-                                                (get-in model k ::not-found)
-                                                (get model k ::not-found))]
-                                  (match? found-v v)))
-                              filter-map))
-                    (vals (deref state)))))))
+      (vec (filter (fn [model]
```

Comment:
Nice one, I forked kibit here and I added the `(vec (filter` to `(filterv` rule.

https://github.com/yogsototh/kibit

Also there is a task about verifying how much code don't have any docstring and lein bikeshed provide such information.

it might be a nice idea to put both kibit and bikeshed in the CI to fail the test if some conditions aren't made. It would help us encode some common coding style rules.